app/models/challenge_claim.rb

```diff
@@ -48,11 +48,11 @@ def ==(other)
   scope :order_by_date, -> { order("created_at ASC") }
 
   def self.order_by_requesting_pseud(dir="ASC")
-    joins(REQUESTING_PSEUD_JOIN).order("pseuds.name #{dir}")
+    joins(REQUESTING_PSEUD_JOIN).order(["pseuds.name ?", dir])
```

Comment:
I believe this also doesn't work, for the same reason as the query in `ChallengeRequestsController`.